inc/profile.class.php

```diff
@@ -940,7 +940,10 @@ function showFormTools($openform=true, $closeform=true) {
                             'field'      => 'project'),
                       array('itemtype'   => 'ProjectTask',
                             'label'      => _n('Task', 'Task', Session::getPluralNumber()),
-                            'field'      => 'projecttask'));
+                            'field'      => 'projecttask'),
+                      array('itemtype'   => 'ProjectTaskTemplate',
```

Comment:
the best is to keep the same behavior, that's right.
If you think a new right should be needed for templates, please open an issue so that we can discuss it :)

thanks.